web/src/settings_org.ts

```diff
@@ -562,6 +609,9 @@ export function discard_realm_property_element_changes(elem) {
             break;
         default:
             if (property_value !== undefined) {
+                // TODO(evy): There are a lot of types that can be in the values of `realm`
+                // and I'm not sure if there's a better thing to do here.
+                assert(typeof property_value === "string" || typeof property_value === "number" || typeof property_value === "boolean");
```

Comment:
i think these three are what we would expect, but i can manually test all the settings at the end of this transition just in case